query/src/test/java/org/infinispan/query/distributed/DistributedMassIndexingTest.java

```diff
@@ -69,6 +73,30 @@ public void testReindexing() throws Exception {
       verifyFindsCar(2, "megane");
    }
 
+   public void testPartiallyReindex() throws Exception {
```

Comment:
This extra method causes lots of failures in the query suite. That's because this test is configured to wipe all the data from the caches after each test, without proper restarting the caches afterwards. So you want to add this to proper cleanup resources:

```
   @Override
   protected boolean cleanupAfterTest() {
      return false;
   }

   @Override
   protected boolean cleanupAfterMethod() {
      return true;
   }

   @BeforeMethod
   @Override
   public void createBeforeMethod() throws Throwable {
      caches.clear();
      super.createBeforeMethod();
   }
```